We do not need to check typing and coverage for this project.

--- argcomplete-3.7.0/pyproject.toml.orig
+++ argcomplete-3.7.0/pyproject.toml
@@ -35,7 +35,7 @@
 register-python-argcomplete = "argcomplete.scripts.register_python_argcomplete:main"
 
 [project.optional-dependencies]
-test = ["coverage", "pexpect", "wheel", "mypy"]
+test = ["pexpect", "wheel"]
 
 [project.urls]
 "Homepage"= "https://github.com/kislyuk/argcomplete"
